main.go

```diff
@@ -41,9 +41,10 @@ func main() {
 	/* Github */
 	funcMap["recentContributions"] = recentContributions
 	funcMap["recentPullRequests"] = recentPullRequests
-	funcMap["recentRepos"] = recentRepos
 	funcMap["recentForks"] = recentForks
 	funcMap["recentReleases"] = recentReleases
+	funcMap["recentRepos"] = recentRepos
+	funcMap["popularRepos"] = popularRepos
```

Comment:
You said you were hesitating about the name

Maybe popularOrgRepos, or popularReposOrg (see https://github.com/charmbracelet/markscribe/pull/5#discussion_r1673076207)

You said it's limited to an organization